examples/beegfs/readme.md

```diff
@@ -0,0 +1,49 @@
+# BeeGFS Cluster
+
+This will deploy a BeeGFS PFS using ephemeral disks (L8s_v2), a headnode, an NFS server running on the headnode (User accounts shared home directories will be stored here), compute cluster and PBS will be deployed.
```

Comment:
can you add the output of azhpcview in the folder?

Reply:
done.